onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -1091,10 +1091,21 @@ def aten_conj_physical(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_constant_pad_nd(self: TensorType, pad: INT64, value: float = 0.0) -> TensorType:
+@torch_op("aten::constant_pad_nd", trace_only=True)
```

Comment:
Why does this need to be trace-only?

Reply:
Thanks for the reminder. I made it with full onnx ops.